Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -273,6 +275,21 @@ protected function getEntity(array $interpretedData)
     }
 
     /**
+     * @param object $entity
+     *
+     * @return ConstraintViolationListInterface
+     */
+    private function validate($entity)
+    {
+        $violationList = $this->validator->validate($entity);
+
+        $event = new PostImportValidationEvent($violationList, $entity);
+        $this->eventDispatcher->dispatch(ImportEvents::POST_VALIDATION, $event);
```

Comment:
So if any listener of this event can actually modify list of validation errors, it's not really a "post" validation event, it's still a part of validation process.